sdk/tables/Azure.Data.Tables/tests/UriQueryParamsCollection.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using System.Text;
+
+namespace Azure.Data.Tables.Tests
+{
+    internal sealed class UriQueryParamsCollection : Dictionary<string, string>
```

Comment:
Would using a regex like `[^\w]sig=[\w\d]+]` be easier here?

Reply:
Probably :) I stole this from storage, but I agree the regex is more succinct.

Reply:
Might be worth pulling this into Shared Source as well if we're copying an existing utility class.  I would rename it though and consider the suggestion to use a regex.

Reply:
deferring to #11778